clap_mangen/src/render.rs

```diff
@@ -29,10 +29,8 @@ pub(crate) fn description(roff: &mut Roff, cmd: &clap::Command) {
     }
 }
 
-pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
-    let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
+pub(crate) fn command_with_args(cmd: &clap::Command, name: String) -> Vec<Inline> {
```

Comment:
Please split out the commits for
- Splitting this function out
- Moving this function below `synopsis` (also, I'm assuming it doesn't need to be `pub(crate)`)